openlibrary/accounts/model.py

```diff
@@ -547,6 +547,27 @@ def xauth(cls, service, test=None, s3_key=None, s3_secret=None, xauth_url=None,
         return simplejson.loads(response)
 
     @classmethod
+    def s3auth(cls, access_key, secret_key):
+        """Authenticates an Archive.org user based on s3 keys"""
+        url = "http://s3.us.archive.org?check_auth=1"
```

Comment:
Not sure how you're testing this model, and this is not required, but seems like maybe this hostname could be split out into separate config so it can be set based on environment, e.g. during testing.

Reply:
great point, will fix (add to olsystem)